scripts/twine_upload.sh

```diff
@@ -29,4 +29,6 @@ python3 -m pip install --upgrade twine wheel setuptools
 
 # Build the distribution and upload.
 python3 setup.py bdist_wheel
+cd contrib/opencensus-common-correlationcontext
+python3 setup.py bdist_wheel --dist-dir ./../../dist/
```

Comment:
Note: this will **not work** since I need to add pushd/popd. I will need a PyPI account and test this on Linux.

I plan to implement a loop, discovering all the sub-folders under contrib and publish them one by one.